metrics/core/labels.go

```diff
@@ -64,6 +64,18 @@ var (
 		Key:         "resource_id",
 		Description: "Identifier(s) specific to a metric",
 	}
+	LabelAcceleratorMake = LabelDescriptor{
+		Key:         "accelerator_make",
```

Comment:
Get rid of this diff. It's changing the labels from what I added in the other PR.

Reply:
I think it is a good thing to make all accelerator related labels `accelerator` prefixed. This may be useful for classification.

Reply:
These labels only show up for accelerator metrics (because we use `acceleratorLabels` only in accelerator metrics). Because of this, I think it is better to not repeat accelerator in the label name. `accelerator_id` is a special case because `id` is usually used for other things.

Also, this will keep things consistent with Stackdriver and prometheus.